docs/source/dataframe-best-practices.rst

```diff
@@ -216,28 +216,25 @@ operation:
 
 For more information see :doc:`Joins <dataframe-joins>`.
 
-Store Data in Apache Parquet Format
------------------------------------
+Use Parquet
+-----------
 
-`Apache Parquet <https://parquet.apache.org/>`_ is a
-columnar binary format that is easy to split into multiple files (easier for
-parallel loading). It is also a common format used by other big data
-systems like `Apache Spark <https://spark.apache.org/>`_ and `Apache Impala
-<https://impala.apache.org/>`_, and so it is useful to interchange with other
-systems:
+`Apache Parquet <https://parquet.apache.org/>`_ is a columnar binary format.
+It is the de-facto standard for the storage of large volumes of tabular data
+and our recommended storage solution for basic tabular data.
 
 .. code-block:: python
 
    df.to_parquet('path/to/my-results/')
    df = dd.read_parquet('path/to/my-results/')
 
-Dask supports reading parquet files with different engine implementations of
```

Comment:
I believe that now we use pyarrow as a default engine, but if you don't have pyarrow install but have fastparquet it'll use fastparquet. Should we be explicit about recommending pyarrow?

Reply:
Na, I'd let the code make this recommendation.  I don't think that we need to specify this in a best practice.